### ISSUE TITLE
PIXI.spine.AnimCache

### ISSUE THREAD

**Issue by ivanpopelyshev** [pixijs/pixi-spine#9]
I think we need to return AnimCache that was in v2. It is useful after all.

**Comment by ivanpopelyshev** [comment]
Currently I use following code:

```javascript
var AnimCache = {};
assetLoader.load(function (loader, resources) {
    for (var key in resources) if (resources.hasOwnProperty(key)) {
        if (resources[key].spineData)
            AnimCache[key] = resources[key].spineData;
    }
    START_THE_GAME(AnimCache);
}
```

**Comment by englercj** [comment]
What benefit does the animCache give you over just passing `resources` to your method?

**Comment by ivanpopelyshev** [comment]
Same as PIXI.utils.TextureCache.

```javascript
stage.add(new Sprite("my_image.png"));
stage.add(new Spine("spine_model.json"));
```

**Comment by englercj** [comment]
FWIW, I hate that sprite syntax and think PIXI.TextureCache is stupid :)

But people love it, and if `Spine.fromAtlas('derp.json')` is something people will want, then go for it. Just keep AnimCache internal (in the parser and internal code) and try to discourage direct use of it.

**Comment by ivanpopelyshev** [comment]
Then I need one more method, both for json and png files that checks if cache has my resource.

```javascript
var obj;
if (PIXI.spine.hasAtlas('derp.json')
   obj = new PIXI.spine.Spine('derp.json');
else
   obj = new PIXI.Sprite('derp.png');
```

Of course use of internal caches is stupid, but its good for fast prototyping. How about adding special flag "storeObjectInCache" in PIXI.loaders.Loader?

**Comment by englercj** [comment]
I wouldn't go that far, I think just doing an AnimCache like TextureCache is OK. Just don't be suprised when I make frowny faces: :frowning:

**Comment by ivanpopelyshev** [comment]
Its ok :+1

**Comment by ivanpopelyshev** [comment]
Ok, i implemented fromAtlas: ceca7ca9d259244eed1854d2762f7333da2b4014
